test1/task1/main.c

```diff
@@ -0,0 +1,124 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <limits.h>
+
+#define NUMBER_OF_BITS sizeof(unsigned int) * 8
+
+enum errorCodes {
```

Comment:
```suggestion
enum ErrorCodes 
{
```
Типы всегда именуются с заглавной (кроме, разве что, стандартной библиотеки)